core/tests/test_utils_test.py

```diff
@@ -156,30 +174,129 @@ def test_call_counter_counts_the_number_of_times_a_function_gets_called(
 
 
 class FailingFunctionTests(test_utils.GenericTestBase):
-    def test_failing_function_fails_for_first_n_calls(self):
+
+    def test_failing_function_never_succeeds_when_n_is_infinity(self):
         class MockError(Exception):
             pass
 
         function = lambda x: x ** 2
 
-        failing_func = test_utils.FailingFunction(function, MockError, 5)
+        failing_func = test_utils.FailingFunction(
+            function, MockError, test_utils.FailingFunction.INFINITY)
 
-        for i in xrange(5):
+        for i in xrange(20):
             with self.assertRaises(MockError):
                 failing_func(i)
-                raise ValueError(str(i))
-
-        self.assertEqual(failing_func(5), 25)
 
-    def test_failing_function_never_succeeds_when_n_is_infinity(self):
```

Comment:
I removed this test as it was not really doing anything -- this function had zero coverage